api/app/serializers/spree/v2/storefront/account_serializer.rb

```diff
@@ -0,0 +1,29 @@
+module Spree
+  module V2
+    module Storefront
+      class AccountSerializer < BaseSerializer
+        set_type :user
+
+        attributes :email, :ship_address_id, :bill_address_id
```

Comment:
@kgorazd we don't need `ship_address_id` and `bill_address_id` they are always visible in relationships node